roles/springboot/tasks/main.yml

```diff
@@ -36,7 +36,7 @@
 - name: "Install Springboot Server services"
   include_tasks: "springboot.yml"
   with_items: "{{ springboot_server_services }}"
-  no_log: true
+    #  no_log: true
```

Comment:
Why is this?

Reply:
Good catch, that was for my local debugging purposes, I've reverted it now.